wp/lib/bap_wp/src/environment.ml

```diff
@@ -477,7 +477,7 @@ let update_stack_base (range : mem_range) (base : int) : mem_range =
 let update_stack_size (range : mem_range) (size : int) : mem_range =
   { range with size = size }
 
-let mk_init_var (env : t) (var : Var.t) : Constr.z3_expr * t =
+let mk_init_var (env : t) (var : Var.t) : Z3.Expr.expr * t =
```

Comment:
Can I ask why you made this change?

Reply:
This is a typo on my end. I had made a major change to `mk_init_var`, however I then ended up reverting `mk_init_var` back to its original version, but forgot to reset the proper types. I'll change the type back to `Constr.z3_expr * t`.